app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,52 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+h1,h2,h3,h4 {
+	color: #24C73A;
+}
+
+body {
+	background-color: #fedf6b;
```

Comment:
Color literals like `#fedf6b` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented with spaces, not tabs